packages/7zip.plugin.asar.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,26 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = "Asar"
+  $pluginsDir = Join-Path ${Env:ProgramFiles} "7-Zip\Formats"
+  New-Item -ItemType Directory -Force -Path $pluginsDir | Out-Null
+
+  $zipUrl = "https://www.tc4shell.com/binary/Asar.zip"
+  $zipSha256 = "ea17751b2d7d607dfc11612e71d0c9d36561e643cdfb2bfb16922a9a0ec6d250"
+
+  $packageArgs = @{
+    packageName    = ${Env:ChocolateyPackageName}
+    unzipLocation  = $pluginsDir
+    url            = $zipUrl
+    checksum       = $zipSha256
+    checksumType   = 'sha256'
+  }
+  Install-ChocolateyZipPackage @packageArgs
+  VM-Assert-Path (Join-Path $pluginsDir "$toolName.64.dll")
+
+  Remove-Item (Join-Path $pluginsDir "$toolName.32.dll") -Force -ea 0
+  Remove-Item (Join-Path $pluginsDir "ReadMe.txt") -Force -ea 0
```

Comment:
in order to support updates and to avoid deleting existent files, I think it would be better to extract the files to a temporary directory and move the needed ones to the `pluginsDir`.

Reply:
Done.